lib/google_places/rectangle.rb

```diff
@@ -0,0 +1,19 @@
+module GooglePlaces
+  class Rectangle
+    def initialize(lat1, lng1, lat2, lng2)
+      @lat1 = ("%.8f"%lat1)
+      @lng1 = ("%.8f"%lng1)
+      @lat2 = ("%.8f"%lat2)
+      @lng2 = ("%.8f"%lng2)
+    end
+
+    def format
+      #SW point of rectangle
+      a1 = [@lat1, @lat2].join (',')
```

Comment:
I would add spaces inside the Array (e.g. `[ @lat1, @lat2 ]`) and remove the space in `join (',')`